src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -390,7 +396,10 @@ export const HeroBody = ({
     initialSectionType
   )
   const { onChange } = useEditableContext()
-  const showNewLayouts = useFeatureIsOn(FEATURE_FLAGS.HOMEPAGE_TEMPLATES)
+  const { blocks } = useFeatureValue(FEATURE_FLAGS.ENABLED_BLOCKS, {
+    blocks: [],
+  })
+  const showNewLayouts = blocks.length > NUM_DEFAULT_HOMEPAGE_BLOCKS
```

Comment:
This code block seems repeated, make this into util func?

Reply:
shure